### ISSUE TITLE
iOS8: Go button should be linked with Next/Submit [2]

### ISSUE THREAD

**Issue by hhonshuku** [kobotoolbox/enketo-express#125]
[Go] button should be linked with [Next/Submit], or [Go] button should not appear so user won't be mislead.

![ios8](https://cloud.githubusercontent.com/assets/8268231/5507975/15a59b52-8776-11e4-9a0b-2ba208f21239.png)

**Comment by enketo-issue-mover** [comment]
This issue was moved to OpenClinica/enketo-express-oc#77
